packages/components/upload/src/use-handlers.ts

```diff
@@ -48,6 +54,7 @@ export const useHandlers = (
     const file = getFile(rawFile)
     if (!file) return
 
+    console.error(err)
```

Comment:
Why add this line?

Reply:
Users can use the `onError` attribute

Reply:
I know. I mean there should be a default behavior of upload failing in Element Plus, since there will be a error icon on file list in ant design and naive-ui.